server/src/main/java/com/sauron/service/BankRequestService.java

```diff
@@ -0,0 +1,10 @@
+package com.sauron.service;
+
+import com.sauron.model.BankRequest;
+
+import java.util.Collection;
+
+public interface BankRequestService {
+
+    <T> Collection<BankRequest<T>> getAllRequests();
```

Comment:
I think this method should be somehow parametrized to get only requests for a certain user and of specific type.

Reply:
This will be handled in future.